clusterloader2/run-e2e.sh

```diff
@@ -34,4 +34,4 @@ if [[ "${DEPLOY_GCI_DRIVER:-false}" == "true" ]]; then
 fi
 
 cd ${CLUSTERLOADER_ROOT}/ && go build -o clusterloader './cmd/'
-./clusterloader --alsologtostderr --v=2 "$@"
+./clusterloader --alsologtostderr --v=2 --node-preload-images=gcr.io/k8s-testimages/probes:latest "$@"
```

Comment:
Let's bump verbosity to `--v=4` to debug what fails now. It looks that ctr pulls images on the nodes, but CL2 time-outs.

Reply:
`W0115 02:29:06.644] I0115 02:29:06.644125   15153 imagepreload.go:133] 0 out of 100 nodes have pulled images`

did the node logs show that the images were pulled successfully?

Reply:
Yes, e.g. node - https://storage.googleapis.com/kubernetes-jenkins/pr-logs/pull/perf-tests/1654/pull-perf-tests-clusterloader2/1349902397828239360/artifacts/e2e-1654-62db2-minion-group-00c3/cl2-image-preload-0.log

It's either that everything works but kubelet needs more time to realize the new image has been pulled and to update the node image or that something doesn't work as it should, e.g. the image is persisted on host or something like that. Let's bump this timeout to 20min to check whether it;s a timeout issue - https://github.com/kubernetes/perf-tests/blob/203cd2740c9eaf8d563998a972d842b1cd0b560f/clusterloader2/pkg/imagepreload/imagepreload.go#L45

Reply:
same issue. shall we add some v4 logs in `hasPreloadedImages`? i'm keen to see what's in `nodeImages`